### PR TITLE
Login & Register

### PR DESCRIPTION
**Pull request by IdjiotSandwiches**

### CODE REVIEW

File: resources/views/component/navigation/navbar.blade.php
Comment (appleaze) [inline resources/views/component/navigation/navbar.blade.php:1]: Navigation to login and register still error

File: resources/views/errors/404.blade.php
Comment (appleaze) [inline resources/views/errors/404.blade.php:1]: Ini masuk lewat mana?

**Review by appleaze — CHANGES_REQUESTED** [review]
Pass certain test case(more on):
https://docs.google.com/spreadsheets/d/1_q1dk5ZqnBbrM0XEZEwz5iAEXABKyBqfMC3KVDjv0-s/edit?gid=1675701385#gid=1675701385

File: resources/views/errors/404.blade.php
Comment (IdjiotSandwiches) [reply resources/views/errors/404.blade.php:1]: ini coba masuk ke route yg g ad aj

File: resources/views/component/navigation/navbar.blade.php
Comment (IdjiotSandwiches) [reply resources/views/component/navigation/navbar.blade.php:1]: bentar gw benerin dlu conflict ny

**Review by appleaze — APPROVED** [review]

**Pull request merged**